### DISCUSSION TITLE
CU401B_S => Fehlende Datenpunkte

### DISCUSSION THREAD

**Discussion by ChrisOertel** [philippoo66/optolink-splitter#62] [Q&A] [answered]
Hallo,
erst einmal Danke für dieses großartige Projekt. Ich habe es am Wochenende einmal ausprobiert und war bei den Datenpunktlisten etwas irritiert, ob diese vollständig sind, bzw. bin ich mir nicht sicher, wie gut die Datenpunkte zwischen Geräten der gleichen Produktart austauschbar sind.
Ich hab eine WP vitoCal 222 mit einer vitotronic Bedieneinheit (Device-ID 204d01430000050c). Damit sollte für mich ja die Datenpunktliste DP_CU401B_S.txt die richtige sein. Beim rumspielen mit der config, habe ich zum experimentieren auch mal eine config für eine vitoCal3xx hier aus dem Forum benutzt und dann gesehen, dass in dieser viele Punkte aus der Liste DP_CU401B_A.txt sind, die aber auch bei mir funktionieren... Z.B.:
    cop_jaz	 0x1680
    cop_jaz_heiz	 0x1681
    cop_jaz_ww	 0x1682
    cop_heiz	 0x1690

Weiß jemand, ob diese Liste vollständig sind, bzw. ob die Datenpunkte zwischen den einzelnen Gerätelisten einfach so übernommen werden können? Oder kann es hier dann doch minimale Unterschiede geben, z.B. ob ein Wert dann nur lesend sein könnte bei einem anderen Gerät aber auch Schreibend?

Danke und frohe Feiertage,
Chris

**Comment by philippoo66** [comment, marked as answer]
moin Chris @ChrisOertel !

ob irgendwelche Datenpunktlisten vollständig und genau passend sind, kann dir wahscheinlich der Weihnachtsstern oder so deuten. Die Chancen, dass der auftaucht, stehen in diesen Tagen recht gut ;-))

leider ist das eben trotz Jahrzehnten Optolink Community immer noch ungeklärt und es hilft letztendlich einfach nur Probieren.

Zum Probieren musst du ja nicht jedesmal deine poll_list ändern, sondern du kannst einfach den Lesebefehl über MQTT (oder auch TCP) absetzen: [Wiki](https://github.com/philippoo66/optolink-splitter/wiki/010-Command-Syntax#command-syntax-for-requests-via-mqtt-and-tcpip) 

Dir auch schöne Festtage & Grüsse!
Phil 

ps. die [xls Tabelle im Wiki](https://github.com/philippoo66/optolink-splitter/wiki/310-Parameter-Addresses) (unten) hast du schon gesehen?
